pylops/basicoperators/__init__.py

```diff
@@ -40,38 +40,38 @@
     SecondDirectionalDerivative     Second Directional derivative.
 """
 
-from .FunctionOperator import *
-from .MemoizeOperator import *
-from .Regression import *
-from .LinearRegression import *
-from .MatrixMult import *
-from .Diagonal import *
-from .Zero import *
-from .Identity import *
-from .Restriction import *
-from .Flip import *
-from .Symmetrize import *
-from .Spread import *
-from .Transpose import *
-from .Roll import *
-from .Pad import *
-from .Sum import *
-from .VStack import *
-from .HStack import *
-from .Block import *
-from .BlockDiag import *
-from .Kronecker import *
-from .Real import *
-from .Imag import *
-from .Conj import *
-from .Smoothing1D import *
-from .Smoothing2D import *
-from .CausalIntegration import *
-from .FirstDerivative import *
-from .SecondDerivative import *
-from .Laplacian import *
-from .Gradient import *
-from .DirectionalDerivative import *
+from .functionoperator import *
+from .memoizeoperator import *
+from .regression import *
+from .linearregression import *
+from .matrixmult import *
+from .diagonal import *
+from .zero import *
+from .identity import *
+from .restriction import *
+from .flip import *
+from .symmetrize import *
+from .spread import *
+from .transpose import *
+from .roll import *
+from .pad import *
+from .sum import *
+from .vstack import *
+from .hstack import *
+from .block import *
+from .blockdiag import *
+from .kronecker import *
+from .real import *
+from .imag import *
+from .conj import *
+from .smoothing1d import *
+from .smoothing2d import *
+from .causalintegration import *
+from .firstderivative import *
+from .secondderivative import *
+from .laplacian import *
+from .gradient import *
+from .directionalderivative import *
 
 __all__ = [
```

Comment:
It is not! But it prevents us from inadvertently exporting functions if we add a file that doesn't have __all__. At the end of the day it's matter of choice, and since it's already there I opted to not remove it.

Reply:
Alright, makes sense. Let's keep it :)